server/tests/e2e/user.e2e-spec.ts

```diff
@@ -1115,6 +1115,45 @@ describe('User', () => {
         });
     });
 
+	describe('PUT /api/user/maps/library/{mapID}', () => {
+		it('should add a new map to the local users library', async () => {
+			const mapLibraryEntry = await global.prisma.mapLibraryEntry.findFirst({
```

Comment:
Some editors will complain here that the type of `global.prisma` is unknown so not know what `mapLibraryEntry` is. So please specific explicitly with `(global.prisma as PrismaService).mapLibraryEntry...`